src/components/UI/Buttons/buttons.js

```diff
@@ -1,12 +1,16 @@
 import "./buttons.css";
 
-function NormalButton(props) { 
+function NormalButton(props) {
     return <button className={`normal-btn ${props.class}`}
-        style={{ backgroundColor: `${props.color}` || "red", padding: `${props.padding}` || "3px",color: `${props.txtcolor}` || "white"  }}
-        onClick={props.click}>
-        {props.name}{props.icon}</button>
+        style={{
+            backgroundColor: `${props.color}` || "red", padding: `${props.padding}` || "3px",
+            color: `${props.txtcolor}` || "white", borderBottom: `${props.borderBottom}`
```

Comment:
Instead of adding classes this way, there's is an npm library called **clsx**, may be you it to apply classes conditionally in a better way

Reply:
okay will check it out